linux_sql/sql/ddl.sql

```diff
@@ -0,0 +1,32 @@
+DROP TABLE host_info, host_usage;
+
+CREATE TABLE PUBLIC.host_info
+  (
+     id               SERIAL NOT NULL,
+     hostname         VARCHAR NOT NULL,
+     cpu_number       INT NOT NULL,
+     cpu_architecture VARCHAR NOT NULL,
+     cpu_model        VARCHAR NOT NULL,
+     cpu_mhz          FLOAT NOT NULL,
+     L2_cache         VARCHAR NOT NULL,
+     total_mem        INT NOT NULL,
+     time_stamp       TIMESTAMP NOT NULL,
+
+     PRIMARY KEY (id),
+     UNIQUE (hostname)
+  );
+
+CREATE TABLE PUBLIC.host_usage
+ (
+     "timestamp"    TIMESTAMP NOT NULL,
```

Comment:
host_usage can have a primary key that consists of a timestamp and host_id, since that is what makes a row unique.